src/autoRespawn/main.ts

```diff
@@ -381,30 +382,61 @@ async function handleMarkChangeEvent(event: MarkChangeEvent) {
       }
 
       const { position } = addedMark
+      const { coalition = addedMark.coalition } = command
+
+      let radius = SPAWNER_MINIMUM_DISPLACEMENT_METERS
+
+      if (command.radius) {
+        radius = command.radius
+      }
 
       // make sure spawner meets minimum displacement requirement
```

Comment:
😅 we can probably remove this comment, it looks like a copy paste (doesn't have to be in this pr, i just noticed it)